src/main/schemas/languages.schema.json

```diff
@@ -1,18 +1,27 @@
 {
   "$schema": "http://json-schema.org/schema#",
-  "$id": "http://isdcf.com/ns/json-schemas/registries/languages/1.0.0-beta.1",
+  "$id": "http://isdcf.com/ns/json-schemas/registries/languages/1.0.0-beta.2",
   "$comment": "Copyright, ISDCF <info@isdcf.com>",
   "title": "Schema for the Languages Registry of the ISDCF",
   "type": "array",
   "items": {
     "type": "object",
     "properties": {
-      "rfc5646Tag": {
-        "type": "string"
+      "comments": {
+        "type": "array",
+        "items": {
+          "type": "string"
+        }
       },
       "dcncTag": {
         "type": "string"
       },
+      "language": {
```

Comment:
"dcncLanguage"

Reply:
updated field name in JSON, schema, and validation